po/it.po

```diff
@@ -0,0 +1,154 @@
+# ITALIAN TRANSLATION.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# ALBANO BATTISTELLA <ALBANOBATTISTELLA@GMAIL.COM>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-02-10 16:59-0300\n"
+"PO-Revision-Date: 2024-03-29 20:17+0100\n"
+"Last-Translator: Albano Battistella <albanobattistella@gmail.com>\n"
+"Language-Team: Italian <LL@li.org>\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:21
+#: /home/moura/Projects/progress-tracker/src/ui/window.cpp:37
+#: src/ui/board-dialog.cpp:24
+msgid "Cancel"
+msgstr "Annulla"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:22
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:50
+#: /home/moura/Projects/progress-tracker/src/ui/preferences-board-dialog.cpp:54
+#: src/ui/board-dialog.cpp:25 src/ui/board-dialog.cpp:53
+msgid "Select a file"
+msgstr "Seleziona un file"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:23
+#: src/ui/board-dialog.cpp:26
+msgid "Select File"
+msgstr "Seleziona file"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:25
+#: src/ui/board-dialog.cpp:28
+msgid "As a file"
+msgstr "Come file"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:26
+#: src/ui/board-dialog.cpp:29
+msgid "As a colour"
+msgstr "Come colore"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:41
+#: src/ui/board-dialog.cpp:44
+msgid "No file selected"
+msgstr "Nessun file selezionato"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-dialog.cpp:55
+#: src/ui/board-dialog.cpp:58
+msgid "Image Files"
+msgstr "File di immagini"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-widget.cpp:15
+msgid "Add List"
+msgstr "Aggiungi elenco"
+
+#: /home/moura/Projects/progress-tracker/src/ui/board-widget.cpp:31
+msgid "New CardList"
+msgstr ""
+
+#: /home/moura/Projects/progress-tracker/src/ui/card.cpp:12
+#: /home/moura/Projects/progress-tracker/src/ui/cardlist-widget.cpp:38
+msgid "Remove"
+msgstr "Rimuovi"
+
+#: /home/moura/Projects/progress-tracker/src/ui/cardlist-widget.cpp:24
+msgid "Add card"
+msgstr ""
+
+#: /home/moura/Projects/progress-tracker/src/ui/cardlist-widget.cpp:140
+msgid "New Card"
+msgstr ""
+
```

Comment:
Those lines are not translating anything. The app will default to the english message